### PR TITLE
Update and enhance README.txt

### PR DESCRIPTION
**Pull request by Gronner**
I added some more information to the readme to make the repository more approachable. In later stages information about building and deploying both front- and backend should be added here as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
-# digitaleswarten
-Teil des Wir Vs Virus Hackathon, Also unsere Idee war im Prinzip digitale Abreissnummern zu haben. Man kreuzt irgendwo auf, registriert sich fuer das Venue in einer Warteschlange und wird notifiziert, wenn man dran ist.
+# WarteFrei
+Teil des [Wir Vs Virus Hackathon](https://wirvsvirushackathon.org/). 
+
+## Die Challenge
+Im Zuge der Ausbreitung des Choronavirus (SARS-CoV-2), werden zusätzliche Patienten deutsche Wartezimmer füllen.
```
Comment (ohaz) [inline README.md]:
```suggestion
Im Zuge der Ausbreitung des Coronavirus (SARS-CoV-2), werden zusätzliche Patienten deutsche Wartezimmer füllen.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+Durch die Versammlung kranker Menschen in den Wartezimmern der Artzpraxen stellen diese jedoch einen ideallen Ansteckungsort für noch nicht infizierte dar.
```
Comment (ohaz) [inline README.md]:
```suggestion
Durch die Versammlung kranker Menschen in den Wartezimmern der Artzpraxen stellen diese jedoch einen idealen Ansteckungsort für noch nicht infizierte dar.
```

Comment (ohaz) [inline README.md]:
```suggestion
Durch die Versammlung kranker Menschen in den Wartezimmern der Artzpraxen stellen diese jedoch einen ideallen Ansteckungsort für noch nicht Infizierte dar.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,21 @@
...
+
+## Die Lösung
+Ziel unsers Projekts *WarteFrei* ist es, das digitale Warten außerhalb des Wartezimmers zu ermöglichen.
+Inspiration hierzu sind die gut bekannten Abreissnummern, wie man sie in deutschen Ämtern oder an Ticketschaltern wiederfindet:
+
+1. Die Arztpraxis hat eine oder mehrere Warteschlangen, in die Patienten eingereiht werden.
+2. Die Patienten erhalten einen Überblick über die aktuelle Wartelage für ihre Warteschlange.
+3. Die Arztpraxis hat die Möglichkeit Patienten zu benachrichtigen, dass sie die Praxis jetzt oder bald betreten können.
+
+Hierdurch soll die Anzahl Personen, die sich in einem Wartezimmer aufhalten reduziert werden oder sogar komplett auf das Warten im Wartezimmer verzichtet werden
```
Comment (ohaz) [inline README.md]:
```suggestion
Hierdurch soll die Anzahl Personen, die sich in einem Wartezimmer aufhalten reduziert werden oder sogar komplett auf das Warten im Wartezimmer verzichtet werden.
```

File: README.md (hunk 1 above)
Comment (yogan) [inline README.md]:
```suggestion
Im Zuge der Ausbreitung des Choronavirus (SARS-CoV-2) werden zusätzliche Patienten deutsche Wartezimmer füllen.
```

File: README.md (hunk 3 above)
Comment (yogan) [inline README.md]:
```suggestion
Hierdurch soll die Anzahl Personen, die sich in einem Wartezimmer aufhalten reduziert werden oder sogar komplett auf das Warten im Wartezimmer verzichtet werden.
```

File: README.md (hunk 1 above)
Comment (Gronner) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (Gronner) [reply README.md]: Fixed

Comment (Gronner) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (Gronner) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (Gronner) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (Gronner) [reply README.md]: Fixed

**Comment by yogan** [comment]
LGTM, feel free to merge.

**Pull request merged**